tests/PySys/monitoring/monitoring_smaller_interval/pysystest.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pysystest type="auto">
+	<description>
+		<title>
+			Validate tedge dm agent messages
```

Comment:
The title could better describe that the test is about sampling interval being less than the batching window.